model-engine/model_engine_server/core/config.py

```diff
@@ -31,7 +31,7 @@
 
 
 @dataclass
-class InfraConfig:
+class _InfraConfig:
```

Comment:
we separate out to base configs to allow us to compose configs without having to worry about default args ordering